AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -41,6 +41,7 @@ @interface ASEditableTextNode () <UITextViewDelegate, NSLayoutManagerDelegate>
 {
   @private
   // Configuration.
+  UITextView *_textView;
```

Comment:
Let's just remove this variable completely, because a) it is not needed, and b) increases the potential for bugs where it is out of sync with the _textKitComponents.